ionoscloud/resource_dbaas_pgsql_database.go

```diff
@@ -0,0 +1,104 @@
+package ionoscloud
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	pgsql "github.com/ionos-cloud/sdk-go-dbaas-postgres"
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/services/dbaas"
+)
+
+func resourceDbaasPgSqlDatabase() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceDbaasPgSqlDatabaseCreate,
+		UpdateContext: resourceDbaasPgSqlDatabaseUpdate,
+		ReadContext:   resourceDbaasPgSqlDatabaseRead,
+		DeleteContext: resourceDbaasPgSqlDatabaseDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceDbaasPgSqlDatabaseImporter,
+		},
+		Schema: map[string]*schema.Schema{
+			"cluster_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				// TODO -- not sure if ForceNew makes sense here since there is no way to modify
+				// the database, so the update method will be empty
+				ForceNew:         true,
```

Comment:
I don't think anyone will want to move their database between clusters if it means re-creating it. We can remove it.

Reply:
@cristiGuranIonos done in cc3d6928b915b5af4bcaf43fbd60d2847e407fa5